include/tag.hpp

```diff
@@ -121,6 +114,32 @@ ENABLE_REFLECTION(fair::graph::tag_t, index, map);
 namespace fair::graph {
 using meta::fixed_string;
 
+void
+merge_maps(const property_map &src, property_map &dest) {
+    for (const auto &[key, value] : src) {
+        if (auto nested_map = std::get_if<pmtv::map_t>(&value)) {
+            // If it's a nested map
+            if (auto it = dest.find(key); it != dest.end()) {
+                // If the key exists in the destination map
+                auto dest_nested_map = std::get_if<pmtv::map_t>(&(it->second));
+                if (dest_nested_map) {
+                    // Merge the nested maps recursively
+                    merge_maps(*nested_map, *dest_nested_map);
+                } else {
+                    // Key exists but not a map, replace it
```

Comment:
Unsure this is the expected behavior of a function called merge. Can you document the reason for the behavior and why it is fine to forget the old value?

Reply:
`update_map` ? Would also work better with the `dest_` parameter name, imho.

Reply:
done... adopted `update_map`